.nvmrc

```diff
@@ -1 +1 @@
-v16.15.1
+v18.19.0
```

Comment:
Would be great if you could change this to a non-Windows line ending.
NVM does not like it. Should be fixable by e.g. opening in Vim, then saving.

Reply:
@fkj It's set to LF endings, which is linux I believe